### PR TITLE
Updating to angular 12.x and ionic 5.x

### PR DESCRIPTION
**Pull request by camerongraybill**
It would be super helpful for us to have this on the up-to-date angular/ionic versions.

Only notable change is I removed some unneeded deps, and I set the peerDeps to have larger ranges.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,44 +12,27 @@
...
     "build": "ng-packagr -p package.json"
   },
-  "author": "Wilson Hobbs",
-  "license": "MIT",
   "peerDependencies": {
-    "@angular/core": "8.2.14",
-    "@ionic/angular": ">=4",
-    "@types/hammerjs": "^2.0.36",
-    "hammerjs": "^2.0.8"
+    "@angular/core": "12.x",
```
Comment (wbhob) [inline package.json:16]: does this syntax actually work? I've seen `^12.0.0` and `>=12.0.0` work, but I have never actually seen this syntax

**Comment by wbhob** [comment]
also – have you been able to test on a local project to make sure these work?

File: package.json (hunk 1 above)
Comment (camerongraybill) [reply package.json:16]: yep! It was new to me too when I was researching peer deps.

source: https://nodejs.org/es/blog/npm/peer-dependencies/

**Comment by camerongraybill** [comment]
> also – have you been able to test on a local project to make sure these work?

I did test it locally with a blank ionic 5/angular 12 app, and the example from the readme and it works.

**Comment by wbhob** [comment]
Excellent - thanks for the help! Merging

**Comment by camerongraybill** [comment]
> Excellent - thanks for the help! Merging

I did not bump the version - can you bump the version and upload a new release? I'd like to start using this.

**Pull request merged**